### ISSUE TITLE
Sending emails

### ISSUE THREAD

**Issue by marceloboeira** [joaocv3/LaTeR#2]
Send emails with a Rails app is not a hard task, but sometimes can be a bit tricky.

TL;DR - Use a 3party mail server with Heroku support for this project, it will avoid a lot of effort on setting up other resources. Remember to setup everything with environment variables, that will help you to change the settings without having to modify/deploy the code again.

I recommend SendGrip or MailChimp, both are great and free! (You don't even need a domain to use it).

[1] https://sendgrid.com/docs/Integrate/Frameworks/rubyonrails.html
[2] http://railsapps.github.io/rails-send-email.html
[3] http://aspiringwebdev.com/e-mail-in-rails-with-mailchimp-and-mandrill-a-comprehensive-guide/
